release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -397,3 +397,122 @@ Issued: 2021-xx-xx
 Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
 
 link:https://access.redhat.com/solutions/<ARTICLE_ID>[{product-title} 4.7.0 container image list]
+
+[id="ocp-4-7-ibm-z"]
+==== IBM Z and LinuxONE
+
+With this release, IBM Z and LinuxONE is now compatible with {product-title} {product-version}. See xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Installing a cluster on IBM Z and LinuxONE] for installation instructions.
+
+With this release, KVM on  {op-system-base} 8.3 or later is supported as a hypervisor for user-provisioned installation of {product-title} {product-version} on IBM Z and LinuxONE.  See xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-ibm-z-kvm[Installing a cluster with {op-system-base} KVM on IBM Z and LinuxONE] for installation instructions.
+
+[discrete]
+===== Restrictions
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
+
```

Comment:
We need to add currently two restrictions:
Bug #1903383 prevents installation on IBM z15 and LinuxONE III. To circumvent this problem on z15 and LinuxONE III please add to the cmdline option dfltcc=off.

Bug #1899762 OCP Nodes installed on FCP based SCSI storage might see unexpected crash. Use DASD base storage for the guest usage till RHEL 8 bug is not closed.

Reply:
@codyhoag I assume this should go in general "Known Issues" section correct? Or would it be better to have IBM Z known issues direclty here?

@ktania46 Please add Holgers restrictions like this (wherever Cody says they belong :-) :
* Currently, installation on IBM z15 and LinuxONE III fails. As a workaround, add the option `dfltcc=off` to `boot.parm` via the command line. ([BZ#1903382](https://bugzilla.redhat.com/show_bug.cgi?id=1903383))

* Currently, OCP nodes installed on FCP-based SCSI storage might crash. As a workaround, use DASD-based storage for the guest usage. ([BZ#1899762](https://bugzilla.redhat.com/show_bug.cgi?id=1899762))

Reply:
@ktania46 Please also add this to the Known Issues:  
* When installing OpenShift Container Platform on IBM Z with KVM, the recommended memory of 8 GB for OCP nodes might not be sufficient. ([BZ#1908875](https://bugzilla.redhat.com/show_bug.cgi?id=1908875))

fyi: @holgwolf